lib/rules/jsx-indent-props.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
 
     schema: [{
       oneOf: [{
-        enum: ['tab']
+        enum: ['tab', 'aligned']
```

Comment:
aligned with what? opening tag, closing tag, first used prop name, tag name, etc?

Reply:
Any better suggestion? Naming is hard :p

My first idea was `tag-aligned` but `jsx-closing-bracket-location` usess that term with a different meaning, and just `tag` is too similar to `tab` IMO.

Reply:
Seems like it's aligned with the first prop?

What happens with:
```jsx
<C a b c
   d
     e
       f
/>
```
Which is properly aligned, d, e, or f?

Reply:
yep, `d`. So yeah, `first-prop` sounds actually a good name.

Just thinking.. should fixing double-space between tag and prop (the stuff from the other comment below) even be part of this rule or a completely new rule? In that case I'd look into creating such a rule and just remove the `afterNameLength !== 1` error here.

Reply:
I think that this rule should only cover prop indentation. I'm not sure what your specific question is tho, could you provide a code example?

Reply:
`<Foo  bar>` --> `<Foo bar>`

So this should then be handled by a separate rule (or is there already one which does that?)

Reply:
Yes, that's not prop indentation - indentation is only "start of line + whitespace + thing"